performance/config/base-config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "browsertime": {
+        "delay": 1
+    },
```

Comment:
Is deze delay nodig?

Reply:
Om het opstarten van de demo-pagina in rekening te brengen.